### ISSUE TITLE
Possibility to skip initial benchmark

### ISSUE THREAD

**Issue by dave7895** [jonathan-laurent/AlphaZero.jl#6]
Is it possible to skip the benchmark without not implementing any?
I am trying to implement the game of Go (with only limited success...) and I am in the need to test the game frequently. Is there any possibility to skip the initial benchmark when only using play?

**Comment by jonathan-laurent** [comment]
If you are just interested in playing against a dumb AI for the sake of testing your game's implementation, I recommend that you run `scripts/mcts.jl` instead of `scripts/alphazero.jl`, after setting the `GAME` environment variable to the name of your game.

Note that I just updated `scripts/mcts.jl` to fix a bug so you may want to do a `git pull` before doing so. Good luck!

**Comment by dave7895** [comment]
Ok thanks. Did not know what it was doing. Working on my remote branch so did not receive the commit automatically and it threw an error. Is working now.
Is there a possibility to profile the agent? It seems to be really slow and I would like to know if the game is so badly implemented, performance wise, or if the agent is somehow really slow.

**Comment by dave7895** [comment]
I tried to use it, but the errror came up that `self_play!` was not defined. Searched the entire project folder for it but could not find anything on a first glance. I found at least one major slowdown and it is now only about 40x slower than tictactoe and not about 40,000x.
And while I tried to train it overnight the was an error `ArgumentError: Categorical: the condition isprobvec(p) is not satisfied.` coming from `rand_categorical(π)`.
